db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model is 'movies'. Consider using a more descriptive name like 'actor_movies' to clearly indicate the relationship direction and avoid potential conflicts.